source/Ship.cpp

```diff
@@ -1490,8 +1491,10 @@ void Ship::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 	DoGeneration();
 
 	// Handle ionization effects, etc.
-	if(ionization)
+	if(ionization || scrambling)
```

Comment:
So if a weapon only did scrambling, we'd try to create ion sparks?
```suggestion
	if(ionization)
```